src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -33,6 +33,10 @@ internal class KnownRoutes
         public const string ExportResourceTypeById = ResourceTypeById + "/" + Export;
         public const string ExportJobLocation = OperationsConstants.Operations + "/" + OperationsConstants.Export + "/" + IdRouteSegment;
 
+        public const string Validate = "$validate";
+        public const string ValidateResourceType = ResourceType + "/" + Validate;
+        public const string ValidateResourceTypeById = ResourceTypeById + "/" + Validate;
```

Comment:
The `ValidateResourceTypeById` route does not seem to be supported currently. Should we support that also?

Reply:
Added